py/kubeflow/testing/create_kf_instance.py

```diff
@@ -85,8 +85,12 @@ def main(): # pylint: disable=too-many-locals,too-many-statements
   # Clean up previous deployment. We are not able to run "kfctl delete all"
   # since we are not able to guarantee apps config in repository is up to date.
   util.run(["rm", "-rf", name], cwd=args.apps_dir)
-  util.run(["gcloud", "deployment-manager", "deployments", "delete", name,
-            "--project", args.project], cwd=args.apps_dir)
+  # TODO(gabrielwen):
```

Comment:
Can you file an issue for this?

Also it's probably better to use the delete_deployment script 
https://github.com/kubeflow/kubeflow/blob/master/scripts/gke/delete_deployment.sh

That script will do a more comprehensive cleanup then just deleting the deployment.

Reply:
Done.